src/test/java/unit/Workspace.java

```diff
@@ -342,4 +342,28 @@ void createFailsWithoutUserFacingId() throws IOException {
         stdErr,
         CoreMatchers.containsString("Missing required option: '--id=<id>'"));
   }
+
+  @Test
+  @DisplayName("workspace create with specified spend profile")
+  void createWorkspaceWithSpendProfile() throws Exception {
+    assumeTrue(Context.getServer().getUserManagerUri() != null);
+
+    // Use the spend owner account
+    TestUser spendProfileUser = TestUser.chooseTestUserWithSpendAccess();
+    spendProfileUser.login();
+
+    // Create the workspace using the default spend profile, verify
+    WorkspaceUtils.createWorkspace(spendProfileUser, Optional.empty());
+    WorkspaceDescription workspaceDescription =
+        WorkspaceManagerService.fromContext().getWorkspace(Context.requireWorkspace().getUuid());
+    assertEquals(
+        UserManagerService.fromContext().getDefaultSpendProfile(/*email=*/ null),
+        workspaceDescription.getSpendProfile());
+
+    // Create the workspace using the alternate spend profile, verify
+    WorkspaceUtils.createWorkspace(spendProfileUser, Optional.empty());
```

Comment:
Should this send the altSpendProfile too?

Reply:
Yes, fixed. Odd that this didnt fail earlier